consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/client/mock/MockClaimStatusUpdateRequestServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.moj.sdt.consumers.client.mock;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.cmc.consumers.client.mock.MockClaimStatusUpdateService;
+import uk.gov.moj.sdt.cmc.consumers.request.ClaimStatusUpdateRequest;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+
+@ExtendWith(MockitoExtension.class)
+public class MockClaimStatusUpdateRequestServiceTest extends AbstractSdtUnitTestBase {
```

Comment:
I think this test class is a duplicate of consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/mock/MockClaimStatusUpdateServiceTest.java.  If it is it can be removed, if not it needs to be finished.

Reply:
deleted